scripts/standardising.py

```diff
@@ -220,7 +220,7 @@ def standardising(
                         [
                             "gdal_footprint",
                             "-t_srs",
-                            EpsgCode.EPSG_4326,
```

Comment:
I don't understand how this could've worked before, because:

```
>>> EpsgCode.EPSG_4326
<EpsgCode.EPSG_4326: 'EPSG:4326'>
>>> str(EpsgCode.EPSG_4326)
'EpsgCode.EPSG_4326'
```

Reply:
Something to deal with `subprocess.run()` I guess?

```
>>> command: list[str]= ["echo", EpsgCode.EPSG_4326]
>>> proc = subprocess.run(command, stdout=subprocess.PIPE, check=True)
>>> print(proc.stdout.decode().strip())
EPSG:4326
```